lib/autoload/course/assessment/programming_test_case_report.rb

```diff
@@ -225,7 +225,11 @@ def messages
     # @return [String]
     def get_meta_attribute(attribute_name)
       meta = @test_case.search('meta')
-      meta.present? && meta[0][attribute_name] ? meta[0][attribute_name] : ''
+      if meta.present? && meta[0][attribute_name]
```

Comment:
I understand the constraints of the GoogleTest Framework, and can read that through the fixtures added. I was suggesting an easier name so that someone without the context of the testcase format (ie. me) can easily understand and do modifications when required.

`get_meta_attribute` originally works cause all the information was stored in the meta tag, but with this change, that is not necessarily true, so it will confuse people when the code says to `get_meta_attribute` when the behavior is `get_meta_attribute_or_test_case_attribute_if_meta_is_missing`.